src/components/Filters/DateRangeFilter.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { fireEvent } from "@testing-library/react";
+import { useState } from "react";
+import { FilterDefs, Filters } from "src/components/Filters";
+import { ProjectFilter, taskCompleteFilter } from "src/components/Filters/testDomain";
+import { click, render, type } from "src/utils/rtl";
+
+describe("DateRangeFilter", () => {
+  it("shows date placeholder text when not given an initial value", async () => {
+    const r = await render(<TestFilters defs={{ dateRange: taskCompleteFilter}} />)
+
+    expect(r.filter_taskCompleted_dateField())
+      .toHaveProperty("placeholder", "This is some placeholder text");
+
+    expect(r.filter_taskCompleted_dateField())
+      .toHaveValue("");
+  });
+
+  it("shows clear btn when a value is present", async () => {
+    const r = await render(<TestFilters defs={{ dateRange: taskCompleteFilter}} />)
+    const dateField = r.filter_taskCompleted_dateField();
+
+    expect(r.queryByTestId("xCircle")).toBeNull();
+    fireEvent.focus(dateField);
+
+    type(dateField, "01/02/20 - 01/19/20")
+
+    fireEvent.blur(dateField);
```

Comment:
Thought: I think our `type` utility already blurs? If you remove this, does the test still pass?

Reply:
It does! Removed the `.blur` & `.focus` events